doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_debugging.rst

```diff
@@ -32,14 +30,19 @@ Debug configurations
 Some applications and samples provide a specific configuration that enables additional debug functionalities.
 You can select custom configurations when you are :ref:`configuring the build settings <cmake_options>`.
 
-Debugging multiple cores
-************************
 
-To debug only the Application core firmware, a single debug session is sufficient.
-To debug the firmware running also on the other cores, you need to set up a separate debug session for each one of the cores you want to debug.
-When debugging another core, the application core debug session runs in the background and you can debug both cores if needed.
+Debugging single-core applications
+**********************************
+
+To debug single-core applications, you can use the ``west debug`` command to start a single debug session with GDB.
+
+Debugging multi-core applications
+*********************************
+
+To debug the firmware running also on cores other than the Application Core, you need to set up a separate debug session for each one of the cores you want to debug.
+When debugging another core, the Application Core debug session runs in the background and you can debug both cores if needed.
 
-If you want to reset the other cores while debugging, make sure to first reset the application core and execute the code.
+If you want to reset the other cores while debugging, make sure to first reset the Application Core and execute the code.
 
 Using GDB as an external debugger
 *********************************
```

Comment:
This is all standard info that should be part of https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/test_and_optimize/debugging.html . Please move it or make a follow-up task.